From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Sun, 30 Oct 2011 22:16:53 +0100
Subject: [PATCH] add FAKEROOT_SYSROOT prefix for relocation

Prefix all paths with FAKEROOT_SYSROOT
It will be replaced by the current sysroot.

Not for upstream.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 scripts/fakeroot.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/fakeroot.in b/scripts/fakeroot.in
index 88ff313cc12a..701e8b3f0888 100755
--- a/scripts/fakeroot.in
+++ b/scripts/fakeroot.in
@@ -30,12 +30,12 @@ fatal ()
 }
 
 # strip /bin/fakeroot to find install prefix
-FAKEROOT_PREFIX=@prefix@
-FAKEROOT_BINDIR=@bindir@
+FAKEROOT_PREFIX="FAKEROOT_SYSROOT@prefix@"
+FAKEROOT_BINDIR="FAKEROOT_SYSROOT@bindir@"
 
 USEABSLIBPATH=@LDPRELOADABS@
 LIB=lib@fakeroot_transformed@@DLSUFFIX@
-PATHS=@libdir@:${FAKEROOT_PREFIX}/lib64/libfakeroot:${FAKEROOT_PREFIX}/lib32/libfakeroot
+PATHS="FAKEROOT_SYSROOT@libdir@":${FAKEROOT_PREFIX}/lib64/libfakeroot:${FAKEROOT_PREFIX}/lib32/libfakeroot
 FAKED=${FAKEROOT_BINDIR}/@faked_transformed@
 
 FAKED_MODE="unknown-is-root"
